src/server/wtr-config.js

```diff
@@ -206,21 +226,33 @@ export class WTRConfig {
 		return config;
 	}
 	getBrowsers(browsers, deviceScaleFactor) {
-		browsers = (this.#requestedBrowsers || browsers || DEFAULT_BROWSERS).map(b => BROWSER_MAP[b] || BROWSER_MAP.chrome);
-
+		browsers = (this.#requestedBrowsers || browsers || DEFAULT_BROWSERS).map(b => {
+			const [product, version] = b.split('-');
+			return BROWSER_MAP[product] ? `${BROWSER_MAP[product]}${version ? `-${version}` : ''}` : BROWSER_MAP.chrome;
+		});
 		if (!Array.isArray(browsers)) throw new TypeError('browsers must be an array');
 
-		return [...new Set(browsers)].map((b) => playwrightLauncher({
-			concurrency: b === BROWSER_MAP.firefox || this.#cliArgs.open ? 1 : undefined, // focus in Firefox unreliable if concurrency > 1 (https://github.com/modernweb-dev/web/issues/238)
-			product: b,
-			createBrowserContext: ({ browser }) => browser.newContext({ deviceScaleFactor, reducedMotion: 'reduce' }),
-			launchOptions: {
-				headless: !this.#cliArgs.open,
-				devtools: false,
-				slowMo: this.#cliArgs.slowmo || 0,
-				args: b === BROWSER_MAP.chrome ? ['--disable-gpu-rasterization'] : []
+		return [...new Set(browsers)].map((b) => {
+			const [product, version] = b.split('-');
+			let revision;
+			if (version) {
+				revision = revisions[product]?.findLast(([gc, v]) => v.startsWith(version))?.[0];
```

Comment:
Avoid `1.3` matching `1.30`.

```suggestion
				revision = revisions[product]?.findLast(([gc, v]) => new RegExp(`${v.replace('.', '\\\.')}(\\.|$)`).test(i))?.[0];
```